### ISSUE TITLE
[Input]:  keyboard navigation issue for input suggestion in JAWS

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents-react#5994]
### Describe the bug

[Input]:  keyboard navigation issue for input suggestion in JAWS
-Can not navigate to all item in input suggestions with keyboard in JAWS. Focus is stuck on one item after navigating few items and JAWS is announcing background element.

### Reproduction steps

1. Open JAWS 2024
2. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-input--docs
3. Go to "Input with customizable SuggestionItem" section
4. enter "a" in input box
5. Start navigating to suggestion items with keyboard

Observed that focus is stuck on 7th item and JAWS started announcing background element. 

### Expected Behaviour

All items should be navigable through keyboard in JAWS. Focus should not be lost.

### UI5 Web Components for React Version

1.29.3

### UI5 Web Components Version

1.24.0

### Browser

Chrome, Edge

### Operating System

Windows 11

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
